x/bank/client/cli_test/helpers.go

```diff
@@ -0,0 +1,46 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/cli_test/helpers"
+	clientkeys "github.com/cosmos/cosmos-sdk/client/keys"
+	"github.com/cosmos/cosmos-sdk/tests"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/stretchr/testify/require"
+)
+
+// TxSend is simcli tx send
+func TxSend(f *helpers.Fixtures, from string, to sdk.AccAddress, amount sdk.Coin, flags ...string) (bool, string, string) {
+	cmd := fmt.Sprintf("%s tx send --keyring-backend=test %s %s %s %v", f.SimcliBinary, from,
+		to, amount, f.Flags())
+	return helpers.ExecuteWriteRetStdStreams(f.T, helpers.AddFlags(cmd, flags), clientkeys.DefaultKeyPass)
+}
+
+// QueryAccount is simcli query account
+func QueryAccount(f *helpers.Fixtures, address sdk.AccAddress, flags ...string) auth.BaseAccount {
+	cmd := fmt.Sprintf("%s query account %s %v", f.SimcliBinary, address, f.Flags())
```

Comment:
nit: add newlines to make this easier to read; it's very hard to grok.

Reply:
Sure, thanks